_delphi_utils_python/tests/test_geomap.py

```diff
@@ -382,6 +382,11 @@ def test_add_geocode(self):
         new_data2 = gmpr.add_geocode(new_data, "state_code", "hhs_region_number")
         assert new_data2["hhs_region_number"].unique().size == 2
 
+        # state_name -> state_id
+        new_data = gmpr.add_geocode(self.zip_data, "zip", "state_name")
+        new_data2 = gmpr.add_geocode(new_data, "state_name", "state_id")
+        assert new_data2.shape == (12, 6)
```

Comment:
should there also be an assert statement for `new_data`?

Reply:
Nah, new_data is just the test data for state_name to state_id, but it starts in zip form